### PR TITLE
Fix building URIs that have only a scheme and an absolute path

### PR DESCRIPTION
**Pull request by csk-ableton**

### CODE REVIEW

**Comment by csk-ableton** [comment]
@glynos Would be nice to get this upstream. Not sure if you forgot about it :)

**Comment by glynos** [comment]
@csk-ableton I thought I added some comments to the review, but I think I got confused by the new interface and they didn't show up. Let me know if you can't see the comments now.

File: test/uri_builder_test.cpp [hunk 1]
```diff
@@ -764,3 +764,12 @@ TEST(builder_test, non_const_non_array_string_literals_should_work) {
     ;
   ASSERT_EQ("http://example.com/foo", builder.uri());
 }
+
+TEST(builder_test, scheme_and_absolute_path) {
```
[the remark this answers is not in the document]
Comment (csk-ableton) [reply test/uri_builder_test.cpp]: `uri::initialize` is called when building URIs from a `ur_builder`. So the code change is related to this test.

**Comment by csk-ableton** [comment]
@glynos I saw them and tried to answer your questions.

**Comment by csk-ableton** [comment]
I added the additional assertions and rebased in d6d0f9b

**Comment by glynos** [comment]
LGTM

**Comment by chfast** [comment]
Can you make a new release with this version please?

**Comment by glynos** [comment]
@chfast done!

https://github.com/cpp-netlib/uri/releases/tag/1.0.1

**Pull request merged**